src/js/media-chrome-listbox.js

```diff
@@ -0,0 +1,278 @@
+import { MediaUIAttributes } from './constants.js';
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+  :host ul {
+    list-style: none;
+  }
+</style>
+<ul tabindex="0" role="listbox">
+  <slot></slot>
+</ul>
+`;
+
+class MediaChromeListbox extends window.HTMLElement {
+  #keysSoFar = '';
+  #clearKeysTimeout = null;
+  #slot;
+  #assignedElements;
```

Comment:
suggestion(non-blocking): If this is likely to be a base class for the dynamic/data driven implementation(s), I suspect some of these likely should not be private (and ECMA doesn't have protected access modifiers 👎)

Reply:
possibly, though, might be useful to have actual getters for things. Like, we probably need a `selectedItem` and `selectedItems` getters and maybe an `options` getter too.

Originally, I was thinking this should stay in PR until we build #282 on top of this, but maybe we can merge this if it's "good enough" but not expose it from `index.js` until that time. That way, we don't have a long-lasting PR but we also don't give folks a half baked listbox unless they explicitly ask for it.